### PR TITLE
Update cluster-property-configuration.adoc - Add reference to updating cluster properties on K8s 

### PR DESCRIPTION
**Pull request by david-yu**
## Description

I had trouble finding how to update cluster properties on K8s, as I was directed from https://docs.redpanda.com/current/reference/properties/cluster-properties/ to https://docs.redpanda.com/current/manage/cluster-maintenance/cluster-property-configuration/. 

## Checks

- [ ] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/manage/pages/cluster-maintenance/cluster-property-configuration.adoc [hunk 1]
```diff
@@ -22,6 +22,8 @@ See also:
 
 == Edit cluster properties
 
+NOTE: For managing cluster properties on Kubernetes, plee see xref:manage:kubernetes/k-cluster-property-configuration.adoc[Configure Cluster Properties in Kubernetes]
```
Comment (micheleRP) [inline modules/manage/pages/cluster-maintenance/cluster-property-configuration.adoc]:
```suggestion
NOTE: For managing cluster properties on Kubernetes, see xref:manage:kubernetes/k-cluster-property-configuration.adoc[Configure Cluster Properties in Kubernetes]
```

**Review by micheleRP — APPROVED** [review]
Thanks, @david-yu!

File: modules/manage/pages/cluster-maintenance/cluster-property-configuration.adoc [hunk 2]
```diff
@@ -22,6 +22,8 @@ See also:
 
 == Edit cluster properties
 
+NOTE: For managing cluster properties on Kubernetes, see xref:manage:kubernetes/k-cluster-property-configuration.adoc[Configure Cluster Properties in Kubernetes]
```
Comment (micheleRP) [inline modules/manage/pages/cluster-maintenance/cluster-property-configuration.adoc]:
```suggestion
NOTE: For managing cluster properties on Kubernetes, see xref:manage:kubernetes/k-cluster-property-configuration.adoc[Configure Cluster Properties in Kubernetes].
```

**Comment by david-yu** [comment]
Please merge if you believe its good to go! I don't have merge rights!

File: modules/manage/pages/cluster-maintenance/cluster-property-configuration.adoc [hunk 3]
```diff
@@ -22,6 +22,8 @@ See also:
 
 == Edit cluster properties
 
+NOTE: For managing cluster properties on Kubernetes, see xref:manage:kubernetes/k-cluster-property-configuration.adoc[Configure Cluster Properties in Kubernetes].
```
Comment (micheleRP) [inline modules/manage/pages/cluster-maintenance/cluster-property-configuration.adoc]:
```suggestion
NOTE: For Kubernetes deployments, see xref:manage:kubernetes/k-cluster-property-configuration.adoc[Configure Cluster Properties in Kubernetes].
```

**Comment by micheleRP** [comment]
> Please merge if you believe its good to go! I don't have merge rights!

Thanks for helping with the docs!

**Pull request merged**